week2/homework/src/index.js

```diff
@@ -1,3 +1,114 @@
 'use strict';
+{
+  const { readList } = require('./readFile.js');
+  const { writeList } = require('./writeFile');
+  const program = require('commander');
 
-// TODO: Write the homework code in this file
+  const todoListPath = './todoList.json';
+
+  function displayHelp() {
+    program
+      .version('0.1.0')
+      .command('node <file-name> ', 'output usage information')
+      .command('list <file-name>', 'list all the items')
+      .command('add <file-name> [item]', 'add item to the list')
+      .command('update <file-name> [index] [new-item]', 'update list')
+      .command('remove <file-name> [index]', 'remove specified item')
+      .command('reset <file-name>', 'reset the list')
+      .parse(process.argv);
+  }
+
+  const initializeTodoList = () => {
+    const todoList = [];
+    writeList(todoListPath, todoList);
+    return todoList;
+  };
+
+  const addTodoListItem = async newTodo => {
+    try {
+      let { todoList } = await readList(todoListPath);
+      if (todoList === undefined) {
+        todoList = initializeTodoList();
+      }
+
+      todoList.forEach((todo, index) => {
```

Comment:
I see this `forEach` is re-assigning all ids to the value they already have. What do you think is the need for this loop?